docs/Translation-translators.md

```diff
@@ -175,6 +175,9 @@ make xx.po
 * Any remaining *obsolete entries* (lines at the end of the file starting 
   with "#~") could be removed. They have no purpose anymore.
 
+* Run `../util/check-msg-args xx.po` to make sure the messages arguments in the
```

Comment:
Why not 'make check-msg-args' instead?

Reply:
`make check-msg-args` checks all the PO files while `../util/check-msg-args xx.po` checks only the relevant file. There shouldn't be errors in the other files but you never know.